srcpkgs/lib3mf/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'lib3mf'
+shopt -s extglob
+
```

Comment:
I think this can cause problems when xbps-src sources multiple templates

Reply:
Alright, switched to environment variables